integration_tests/features/Reorgs.feature

```diff
@@ -166,48 +172,3 @@ Scenario Outline: Massive multiple reorg
         | X1     | Y1     | X2    | Y2   | SYNC_TIME |
         | 100    | 125    | 150   | 175  | 30        |
         | 1010   | 1110   | 1210  | 1310 | 60        |
-
-Scenario Outline: Massive reorg simple case
```

Comment:
Where did this test go?

Reply:
This test was only done initially because `Massive multiple reorg` failed. 
`Massive reorg simple case` can now be removed because it is wholly taken up in `Massive multiple reorg`; just repeating a portion of that test.